### PR TITLE
[#1234] Bug/User with no Company

### PR DESCRIPTION
**Pull request by wanyaland**
- Fixed issue raised when user with no organization is assigned cla-manager-designee for a given organization

Signed-off-by: wanyaland <wanyaland@gmail.com>

### CODE REVIEW

File: cla-backend-go/v2/cla_manager/service.go [hunk 1]
```diff
@@ -518,6 +520,13 @@ func (s *service) CreateCLAManagerRequest(contactAdmin bool, companyID string, p
 		return nil, ErrNotInOrg
 	}
 
+	// Check if user has organization
+	if lfxUser.Account.Name == "" {
+		msg := fmt.Sprintf("ACS service - 400 Bad Request - User: %s is not associated with any organization", userEmail)
```
Comment (dealako) [inline cla-backend-go/v2/cla_manager/service.go:525]: Even though this is a potential ACS issue, we want to report the error as 'EasyCLA Service' so the caller knows where the error originated.

File: cla-backend-go/v2/cla_manager/service.go [hunk 2]
```diff
@@ -57,6 +57,8 @@ var (
 	ErrNoOrgAdmins = errors.New("no Admins in company ")
 	//ErrRoleScopeConflict thrown if user already has role scope
 	ErrRoleScopeConflict = errors.New("role or scope conflict")
+	//ErrNotOrgAssociated thrown if user is not associated with org
+	ErrNotOrgAssociated = errors.New("user is not associaeted with organization")
```
Comment (dealako) [inline cla-backend-go/v2/cla_manager/service.go:61]: In this PR, below you use a different error message: ErrNotInOrg.  Is this error message used anywhere?

File: cla-backend-go/v2/cla_manager/service.go [hunk 3]
```diff
@@ -518,6 +520,13 @@ func (s *service) CreateCLAManagerRequest(contactAdmin bool, companyID string, p
 		return nil, ErrNotInOrg
 	}
 
+	// Check if user has organization
+	if lfxUser.Account.Name == "" {
```
Comment (dealako) [inline cla-backend-go/v2/cla_manager/service.go:524]: Is this how we check that the specified user is not associated with any organization?

Comment (wanyaland) [reply cla-backend-go/v2/cla_manager/service.go:524]: So on checking the Lfx User Model this is the association with company via the Account attribute. I could be wrong

File: cla-backend-go/v2/cla_manager/service.go (hunk 1 above)
Comment (wanyaland) [reply cla-backend-go/v2/cla_manager/service.go:525]: > Even though this is a potential ACS issue, we want to report the error as 'EasyCLA Service' so the caller knows where the error originated.

👍

**Pull request merged**